sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -24,6 +24,9 @@ public class CertificateClient
     {
         internal const string CertificatesPath = "/certificates/";
         internal const string DeletedCertificatesPath = "/deletedcertificates/";
+        private const string OTelCertificateNameKey = "az.key_vault.certificate.name";
```

Comment:
I wonder whether we should colloquially call this `keyvault`. In places where it's one word e.g., `az` it's just `keyvault`. Of course, if so, we should do this throughout.

Reply:
good point, changed to `az.keyvault.*` (`az.` is a common prefix we use literally everywhere on azure-specific attributes and it feels good to keep it)

Reply:
The `az.` prefix is fine. I was referring to the `az` CLI e.g., `az keyvault {subcommand} ...`.

Reply:
great suggestion! The CLI is not always perfect (things like question-answering doesn't have CLI), but I ended up using the path we have in the repos to group SDKs by. E.g. question-answering becomes `cognitivelanguage` and so on. This should give us consistency. 

Will document naming conventions in scope of https://github.com/Azure/azure-sdk-for-net/issues/39785